### ISSUE TITLE
Docker: Container Unable to Start

### ISSUE THREAD

**Issue by deedubb** [EMCECS/ECS-CommunityEdition#147]
I was able to complete the steps on https://github.com/EMCECS/ECS-CommunityEdition/blob/master/Documentation/ECS-SingleNode-Instructions.md but during my investigation of why I couldn't save a 600MB file into an s3 bucket I rebooted the host VM. Since then I can't access my container again.

I would appreciate any assistance.

```
[root@ecs1 ecs-single-node]# docker start -ai 074ec02709a9
06272016 08:31:17 : ############################### boot operation begin ############################
06272016 08:31:17 : checking 'which' command
/opt/vipr/boot/boot_datasvcs_lib.sh: line 31: /var/log/boot.log: Permission denied
/opt/vipr/boot/boot_datasvcs_lib.sh: line 31: /var/log/boot.log: Permission denied
which is /usr/bin/which
06272016 08:31:17 : exporting environment variable VIPR_DEPLOYMENT_TYPE to fabric
/opt/vipr/boot/boot_datasvcs_lib.sh: line 31: /var/log/boot.log: Permission denied
rsyslogd: pidfile '/var/run/rsyslogd.pid' and pid 15 already exist.
06272016 08:32:17 : Parsing /host/data/network.json
/opt/vipr/boot/boot_datasvcs_lib.sh: line 31: /var/log/boot.log: Permission denied
/opt/vipr/boot/boot_datasvcs_lib.sh: line 31: /var/log/boot.log: Permission denied
/opt/vipr/boot/boot_datasvcs_lib.sh: line 31: /var/log/boot.log: Permission denied
06272016 08:32:17 : VERSION: 2.2.1.0.77331.4f57cc6 private iface: 06272016 08:32:17 : file /host/data/network.json not found public iface: 06272016 08:32:17 : file /host/data/network.json not found START VIPR:
06272016 08:32:17 : expecting data partitions to be mounted under /dae
/opt/vipr/boot/boot_datasvcs_lib.sh: line 31: /var/log/boot.log: Permission denied
/opt/vipr/boot/boot_datasvcs_lib.sh: line 31: /var/log/boot.log: Permission denied
06272016 08:32:17 : StorageServer - setting partitionroot=/dae in /opt/storageos/conf/storageserver.conf
/opt/vipr/boot/boot_datasvcs_lib.sh: line 31: /var/log/boot.log: Permission denied
06272016 08:32:17 : StorageServer JSON configuration file is /data/partitions.json
/opt/vipr/boot/boot_datasvcs_lib.sh: line 31: /var/log/boot.log: Permission denied
/opt/vipr/boot/boot_datasvcs_lib.sh: line 139: /data/partitions.json: Permission denied
06272016 08:32:17 : validating config.iso and fabric version file
06272016 08:32:17 : config.iso is missing in the /data location
/opt/vipr/boot/boot_datasvcs_lib.sh: line 31: /var/log/boot.log: Permission denied
/opt/vipr/boot/boot_datasvcs_lib.sh: line 31: /var/log/boot.log: Permission denied
06272016 08:32:17 : fetching network information
/opt/vipr/boot/boot_datasvcs_lib.sh: line 31: /var/log/boot.log: Permission denied
/opt/vipr/boot/boot_datasvcs_lib.sh: line 31: /var/log/boot.log: Permission denied
/opt/vipr/boot/boot_datasvcs_lib.sh: line 31: /var/log/boot.log: Permission denied
/opt/vipr/boot/boot_datasvcs_lib.sh: line 31: /var/log/boot.log: Permission denied
06272016 08:32:17 : Data IP address of the machine: 06272016 08:32:17 : file /host/data/network.json not found
/opt/vipr/boot/boot_datasvcs_lib.sh: line 31: /var/log/boot.log: Permission denied
06272016 08:32:17 : Management IP address of the machine: 06272016 08:32:17 : file /host/data/network.json not found
06272016 08:32:17 : Geo IP address of the machine: 06272016 08:32:17 : file /host/data/network.json not found
/opt/vipr/boot/boot_datasvcs_lib.sh: line 31: /var/log/boot.log: Permission denied
/opt/vipr/boot/boot_datasvcs_lib.sh: line 31: /var/log/boot.log: Permission denied
06272016 08:32:: error fetching interface information: Device not found
06272016 08:32:18 : NETMASK found:
06272016 08:32:18 : netmask could not be found from 06272016 08:32:17 : file /host/data/network.json not found
/opt/vipr/boot/boot_datasvcs_lib.sh: line 31: /var/log/boot.log: Permission denied
/opt/vipr/boot/boot_datasvcs_lib.sh: line 31: /var/log/boot.log: Permission denied

```

**Comment by APeter0807** [comment]
Definitely sounds like you managed to start the container as a non-root user; does the issue persist if you start without the attach/interactive options?

**Comment by deedubb** [comment]
Yes

**Comment by deedubb** [comment]
Updated **/etc/sysconfig/selinux** set `SELINUX=permissive`. Default is now enforcing.
Updated **/etc/sysconfig/docker** set `OPTIONS='--log-driver=journald'` instead of OPTIONS='--selinux-enabled --log-driver=journald' 
Ran`chmod +x /etc/rc.d/rc.local`
Rebooted. 
Now everything is rebooting properly

**Comment by jasoncwik** [comment]
There's a better solution here.  In Docker 1.7+ you can tag the volumes with 'z' to tell Docker to tag the volumes so SELinux can allow Docker to access the volumes after reboot.

**Comment by jasoncwik** [comment]
Released as part of 3.0.0.1
